src/config.ts

```diff
@@ -122,10 +121,8 @@ export const ALCHEMY_BASE_API_KEY = keyOrUndefined(
 )
 
 export const ZENDESK_API_KEY = keyOrUndefined(secretsFile, DEFAULT_TESTNET, 'ZENDESK_API_KEY')
-export const STATSIG_API_KEY =
-  keyOrUndefined(secretsFile, DEFAULT_TESTNET, 'STATSIG_API_KEY') ??
-  // dummy key as fallback for e2e tests, which use local mode
-  'client-key'
+export const STATSIG_API_KEY = keyOrUndefined(secretsFile, DEFAULT_TESTNET, 'STATSIG_API_KEY')
+export const STATSIG_ENABLED = !isE2EEnv && !!STATSIG_API_KEY
```

Comment:
Kept `isE2EEnv` here as when running locally, the secrets are decrypted, but we should still keep statsig disabled.

I think with the move to the runtime we'll update "secrets" management and will just use env variables. So this special case of e2e env will not be needed anymore.